templates/zerver/help/deactivate-or-reactivate-a-bot.md

```diff
@@ -1,7 +1,9 @@
 # Deactivate or reactivate a bot
 
-Zulip organization administrators have the ability to deactivate or reactivate any bots
-in their organization.
+{!admin-only.md!}
```

Comment:
This page should probably link to the fact that any user can manage their own bots on the "Your bots" subpage of settings.